providers/tests/databricks/hooks/test_databricks.py

```diff
@@ -1222,6 +1229,8 @@ class TestDatabricksHookTokenInPassword:
     def setup_method(self, method, session=None):
         conn = session.query(Connection).filter(Connection.conn_id == DEFAULT_CONN_ID).first()
         conn.host = HOST
+        conn.schem = None
```

Comment:
static checks fail on this line 
```suggestion
        conn.schema = None
```

Reply:
ah, my bad! thanks for pointing it out